### PR TITLE
Point at support community on Discord

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

**Comment by oxarbitrage** [comment]
Nice one. What about changing the icon of chat support to:  :speech_balloon: ?

**Comment by rex4539** [comment]
I like the idea. If others agree, I can change that too.

**Comment by str4d** [comment]
Icon change sounds good to me.

**Comment by rex4539** [comment]
Icon changed.

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Please see our [user guide](https://zcash.readthedocs.io/en/latest/rtd_pages/rtd
 * :blue_book: See the documentation at the [ReadTheDocs](https://zcash.readthedocs.io)
   for help and more information.
 * :incoming_envelope: Ask for help on the [Zcash](https://forum.z.cash/) forum.
-* :mag: Chat with our support community on [Rocket.Chat](https://chat.zcashcommunity.com/channel/user-support)
+* :speech_balloon: Chat with our support community on [Discord](https://discordapp.com/channels/669694001464737815/671029188353851393/)
```
Comment (str4d) [inline README.md:46]: I confirmed that this link points to the `#tech-support` channel in the ZcashCommunity Discord.

**Comment by str4d** [comment]
@zkbot r+

**Comment by str4d** [comment]
Transient failures.

@zkbot retry

**Comment by str4d** [comment]
@zkbot retry

**Comment by str4d** [comment]
@zkbot retry

**Comment by str4d** [comment]
This is ridiculous - four transient failures in a row. I'm overriding the CI merge policy for this specific PR, as it is purely a `README` fix.

**Pull request merged**